Older versions of bash do not support the =~ construct, so stop
using it.
---
 quilt/mail.in |   10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

--- quilt-0.48.orig/quilt/mail.in	2009-01-31 03:28:06.000000000 +0100
+++ quilt-0.48/quilt/mail.in	2009-06-10 14:24:08.000000000 +0200
@@ -101,10 +101,14 @@ references_header() {
 	then
 		in_reply_to=$(formail -x In-Reply-To: < "$message")
 		in_reply_to=${in_reply_to# }
-		if [ -n "$in_reply_to" ] &&
-		   ! [[ "$in_reply_to" =~ "@.*@" ]]
+		if [ -n "$in_reply_to" ]
 		then
-			references=$in_reply_to
+			case "$in_reply_to" in
+			*@*@*)
+				;;
+			*)	references=$in_reply_to
+				;;
+			esac
 		fi
 	fi
 	if [ -z "$references" ]
